tests/config/tags.yml

```diff
@@ -433,6 +433,30 @@ mutect2:
   - tests/csv/3.0/recalibrated_tumoronly.csv
   - tests/test_mutect2.yml
 
+## Sentieon
```

Comment:
Will this not cause the pytests for sentieon to be run twice?

Reply:
agree, we don't need that to run twice.
Maybe it was added just to trigger it